src/VisualStudio/Core/Def/ProjectSystem/MetadataReferences/VisualStudioMetadataReferenceManager.cs

```diff
@@ -303,34 +303,24 @@ bool TryGetFileMappingFromMetadataImporter(FileKey fileKey, [NotNullWhen(true)]
         FileKey fileKey,
         Func<FileKey, (ModuleMetadata moduleMetadata, TemporaryStorageStreamHandle? storageHandle)> moduleMetadataFactory)
     {
-        var (manifestModule, manifestHandle) = moduleMetadataFactory(fileKey);
```

Comment:
the core bug was that we were not doing anything with this manifestHandle in the case where this was a *multi-module* assembly.  I'll doc how this happened.

First, we would get the handle here.